_posts/2016-04-01-fintech-demo-day.md

```diff
@@ -0,0 +1,153 @@
+For example, there are more than sixty red flags for older populations and
+EverSafe's technology scans its users' financial accounts looking for these red
+flags. Family members get alerted as suspicious activity arises.
+
+[Twitter](https://twitter.com/EverSafeSeniors)
+
+### ForwardLane
+
+[ForwardLane](http://forwardlane.com/home.html) offers ultra high net worth investors customized investment
+advice. They spoke about the $800bn shortfall for the 42mm ultra affluent
+investors who are affected by market volatility. They attempt to solve this
+problem by programmatically generating questions and advice for their users
+through techniques in quantitative finance and machine learning.
+
+[Twitter](https://twitter.com/Forward_Lane)
+
+### Kyper
+
+[Kyper](https://www.kyper.com/)'s marketplace matches data providers with
+developers. Similar to [Quandl](https://www.quandl.com/), it matches developers
+with much needed datasets that allow them to quickly prototype and develop
+ideas. They are trying to solve the problem of imposed technologies by data
+providers, like Java, .NET, C#, various communication protocols, formats like
+csv, xml, or xls. Their marketplace allows the developer to gain access to data
+based on their needs rather than the imposed restrictions of the data providers.
+
+[Twitter](https://twitter.com/kyperdata)
+
+### Nutonian
+
+[Nutonian](http://nutonian.com)'s main product Eureqa stems from a Cornell
+University AI Lab. It helps investment professionals analyze data by providing
+the best models they can use based on billions of datapoints. According to them,
+there are hundreds of thousands of "Data scientist" job openings out there,
+which indicates that there is a shortage of expertise out there to help with
+good analytics. This is the general process of their product: (1) User either
+uploads data or connects to a data provider (like Factset); (2) The data gets
+analyzed against thousands of economic indicators; and (3) A series of models
+and their historical accuracy is sent back to the user, from which they can
+decide which are the best to use.
+
+[Twitter](https://twitter.com/nutonian)
+
+### Open Invest Co.
+
+[Open Invest](https://openinvest.co/) gives its users the ability to choose
+investment strategies based on both financial return and social good. Their
+argument is that Millenials invest based on their own values, and that
+investing with an eye for social good can be just as profitable as not. They
+offer a clean, crisp dashboard that lets users pick industries they are not
+interested in (like Tobacco, for example), and their software returns possible
+investment strategies based on their inputs. The system builds on itself as it
+gains more insight in what types of companies its users don't want to invest in.
+
+[Twitter](https://twitter.com/openinvestco)
+
+### Trigger
+
+[Trigger](http://www.triggerfinance.com)'s users can create a simple `IF x THEN y` type of statement
```

Comment:
I think it should be ok. The blog treated the in-line formatting in the SEC parser post fine (but missed out on the multi-line elixir formatting):

![screen shot 2016-04-01 at 11 16 07 am](https://cloud.githubusercontent.com/assets/7072744/14211297/39642864-f7fb-11e5-907b-956860c77d64.png)